com.example.WebPConverter.json

```diff
@@ -0,0 +1,62 @@
+{
+  "app-id": "com.example.WebPConverter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "webp_converter.py",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
+    "--filesystem=xdg-config/gtk-3.0:ro",
+    "--filesystem=xdg-config/gtk-4.0:ro",
+    "--talk-name=org.freedesktop.portal.Desktop",
+    "--own-name=com.example.WebPConverter"
+  ],
+  "extensions": {
+    "org.gtk.Gtk3theme": {
+      "version": "3.22"
+    },
+    "org.gtk.Gtk4theme": {
+      "version": "4.0"
+    }
+  },
```

Comment:
I don't know which bit of the documentation told you to add this but it's definitely wrong.

Reply:
I added this as it was the only way to get the app to play nice with custom themes. I have removed it for now.